index.html

```diff
@@ -34,13 +55,51 @@
 
         #notification {
             position: fixed;
-            bottom: 20px;
-            right: 20px;
-            padding: 10px;
-            background: #28a745;
+            bottom: calc(var(--spacing-unit) * 2.5);
+            right: calc(var(--spacing-unit) * 2.5);
+            padding: var(--spacing-unit);
+            background: var(--success-color);
             color: white;
-            border-radius: 4px;
+            border-radius: var(--border-radius);
             display: none;
+            animation: slideIn 0.3s ease-out;
+        }
+
+        @keyframes slideIn {
+            from { transform: translateX(100%); }
+            to { transform: translateX(0); }
+        }
+
+        .filters {
+            padding: calc(var(--spacing-unit) * 2);
+            background: #f8f9fa;
+            margin-bottom: calc(var(--spacing-unit) * 2);
+            border-radius: var(--border-radius);
+        }
+
+        .btn-group {
+            display: flex;
+            gap: var(--spacing-unit);
+        }
+
+        #cartCount {
+            position: fixed;
+            top: calc(var(--spacing-unit) * 2);
+            right: calc(var(--spacing-unit) * 2);
+            background: var(--primary-color);
+            color: white;
+            padding: var(--spacing-unit);
+            border-radius: var(--border-radius);
+        }
+
+        .price-filter {
+            margin-top: calc(var(--spacing-unit) * 2);
```

Comment:
Consider adding a max-width here to prevent the range input from stretching too wide on larger screens.